internal/tools/orchestrator/scheduler/executor/plugins/k8s/daemonset/daemonset.go

```diff
@@ -15,34 +15,30 @@
 package daemonset
 
 import (
-	"bytes"
+	"context"
 	"encoding/json"
 	"fmt"
-	"net/url"
 	"strings"
 
+	"github.com/pkg/errors"
 	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
 
-	"github.com/erda-project/erda/internal/tools/orchestrator/scheduler/executor/plugins/k8s/k8sapi"
 	"github.com/erda-project/erda/internal/tools/orchestrator/scheduler/executor/plugins/k8s/k8serror"
-	"github.com/erda-project/erda/pkg/http/httpclient"
+	k8stypes "github.com/erda-project/erda/internal/tools/orchestrator/scheduler/executor/plugins/k8s/types"
```

Comment:
Maintain the use of 'types' for references within the component; use 'k8stypes' to identify `k8s.io/apimachinery/pkg/types` similar to 'k8serrors.